komodor/manifest.json

```
@@ -0,0 +1,69 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "995fe904-e761-4f2f-8dbf-148baf3f080a",
+  "app_id": "komodor",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+	"Category::Troubleshooting"
```

I think the validation check [is complaining](https://github.com/DataDog/integrations-extras/runs/7026162900?check_suite_focus=true#step:10:37) about a comma missing here:

```suggestion
    "Category::Troubleshooting",
```